chemicaltoolbox/autodock_vina/docking/test-data/NuBBE_1_obabel_3D_-_3u1i_for_DM.sdf

```diff
@@ -0,0 +1,1620 @@
+    9  A    between atoms: C_19  and  O_20 
+
+>  <TORSDO>
+F 9
+
+>  <SCORE>
+0.0
+
+>  <RMSD_LB>
+5.737
+
+>  <RMSD_UB>
+8.627
+
+$$$$
+./output1.dat
+ OpenBabel06171915303D
+
+ 23 23  0  0  0  0  0  0  0  0999 V2000
+   66.6420   67.4990   40.1730 O   0  0  0  0  0  0  0  0  0  0  0  0
+   66.8380   68.6440   40.5360 C   0  0  0  0  0  0  0  0  0  0  0  0
+   67.8530   69.1000   41.3080 O   0  0  0  0  0  0  0  0  0  0  0  0
+   69.1820   68.8470   40.8480 C   0  0  0  0  0  0  0  0  0  0  0  0
+   69.6150   70.0630   40.0810 C   0  0  0  0  0  0  0  0  0  0  0  0
+   69.6800   70.1970   38.7400 C   0  0  0  0  0  0  0  0  0  0  0  0
+   69.3200   69.1030   37.7690 C   0  0  0  0  0  0  0  0  0  0  0  0
+   70.1450   71.5000   38.1080 C   0  0  0  0  0  0  0  0  0  0  0  0
+   70.0730   72.6890   39.0730 C   0  0  0  0  0  0  0  0  0  0  0  0
+   68.8480   73.5420   38.8600 C   0  0  0  0  0  0  0  0  0  0  0  0
+   67.8250   73.7350   39.7180 C   0  0  0  0  0  0  0  0  0  0  0  0
+   67.6850   73.0530   41.0510 C   0  0  0  0  0  0  0  0  0  0  0  0
+   66.7000   74.6830   39.3890 C   0  0  0  0  0  0  0  0  0  0  0  0
+   65.9480   69.7870   40.2070 C   0  0  0  0  0  0  0  0  0  0  0  0
+   65.0990   70.3450   41.1740 C   0  0  0  0  0  0  0  0  0  0  0  0
+   64.2740   71.4040   40.8210 C   0  0  0  0  0  0  0  0  0  0  0  0
+   64.2960   71.9130   39.5250 C   0  0  0  0  0  0  0  0  0  0  0  0
+   65.1460   71.3820   38.5650 C   0  0  0  0  0  0  0  0  0  0  0  0
+   65.9820   70.3160   38.9060 C   0  0  0  0  0  0  0  0  0  0  0  0
+   63.4210   71.9850   41.7190 O   0  0  0  0  0  0  0  0  0  0  0  0
+   62.7920   72.6460   41.4570 H   0  0  0  0  0  0  0  0  0  0  0  0
+   63.4750   72.9500   39.1890 O   0  0  0  0  0  0  0  0  0  0  0  0
+   62.7330   72.7990   38.6140 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  2  0  0  0  0
+  2  3  1  0  0  0  0
+  4  3  1  0  0  0  0
+  5  4  1  0  0  0  0
+  6  5  2  0  0  0  0
+  7  6  1  0  0  0  0
+  8  6  1  0  0  0  0
+  8  9  1  0  0  0  0
+ 10  9  1  0  0  0  0
+ 10 11  2  0  0  0  0
+ 11 12  1  0  0  0  0
+ 13 11  1  0  0  0  0
+ 14  2  1  0  0  0  0
+ 14 15  2  0  0  0  0
+ 16 15  1  0  0  0  0
+ 16 20  1  0  0  0  0
+ 17 16  2  0  0  0  0
+ 18 19  2  0  0  0  0
+ 18 17  1  0  0  0  0
+ 19 14  1  0  0  0  0
+ 21 20  1  0  0  0  0
+ 22 17  1  0  0  0  0
+ 23 22  1  0  0  0  0
+M  END
+>  <MODEL>
+20
+
+>  <REMARK>
+ VINA RESULT:       0.0      2.715      4.755
+  9 active torsions:
+  status: ('A' for Active; 'I' for Inactive)
+    1  A    between atoms: C_2  and  O_3 
+    2  A    between atoms: C_2  and  C_14 
+    3  A    between atoms: O_3  and  C_4 
+    4  A    between atoms: C_4  and  C_5 
+    5  A    between atoms: C_6  and  C_8 
+    6  A    between atoms: C_8  and  C_9 
+    7  A    between atoms: C_9  and  C_10 
+    8  A    between atoms: C_16  and  O_17 
+    9  A    between atoms: C_19  and  O_20 
+
+>  <TORSDO>
+F 9
+
+>  <SCORE>
```

Comment:
all scores are 0, is that expected?

Reply:
Yes, it is also 0 in the log file: https://github.com/bgruening/galaxytools/blob/5cf8348a6166b100df2efd4cadf9fb08419a6b52/chemicaltoolbox/autodock_vina/docking/test-data/NuBBE_1_obabel_3D_-_3u1i_for_DM.log